DemoqaProject/tests/CheckBoxTest.cs

```diff
@@ -0,0 +1,28 @@
+using DemoqaProject.PageObjects;
+using NUnit.Framework;
+
+namespace DemoqaProject
+{
+    public class CheckBoxTest : BaseSeleniumTest
+    {
+        [Test]
+        public void SelectAll()
+        {
+            HomePage homePage = new HomePage();
+            homePage.NavigateToElementPage();
+            Elements elements = new Elements();
+            elements.NavigateToCheckBox();
+
+            Thread.Sleep(3000);
+            elements.Count();
+            Thread.Sleep(3000);
+            Assert.IsFalse(elements.selectAll.Selected);
+            
+            elements.SpecificCategories();
+            elements.SelectGroupAndUncheckSub();
+            elements.ExpendAndCollapseAll();
```

Comment:
These are not asserts, they are not checking anything, just some actions.

Reply:
Here, I can't understand why should I check method? Please give me one example how should I do here.